dm/openapi/spec/dm.yaml

```diff
@@ -2168,13 +2168,14 @@ components:
           description: whether to remove meta database in downstream database
         source_name_list:
           $ref: "#/components/schemas/SourceNameList"
-        # start_time:
-        #   type: string
-        #   example: "2006-01-02 15:04:05"
-        #   description: task start time
-        # safe_mode_time_duration:
-        #   example: "1s"
-        #   description: time duration of safe mode
+        start_time:
+          type: string
+          example: "2006-01-02 15:04:05"
+          description: task start time
+        safe_mode_time_duration:
+          type: string
+          example: "10s"
+          description: time duration of safe mode
```

Comment:
we also support `WaitTimeOnStop` in cliArgs now

Reply:
`WaitTimeOnStop` is not in the requirement, maybe i will add later pr and add some integration test.